app/build.gradle

```diff
@@ -145,15 +145,13 @@ dependencies {
     compile 'com.google.android.gms:play-services-appstate:6.5.87'
 
     //Google
-    compile 'com.google.code.gson:gson:2.3'
+    compile 'com.google.code.gson:gson:2.3.1'
     compile 'com.google:volley:4.3'
-    compile 'com.google.http-client:google-http-client-gson:1.18.0-rc'
+    compile 'com.google.http-client:google-http-client-gson:1.19.0'
     compile 'com.google.apis:google-api-services-plus:v1-rev126-1.18.0-rc'
```

Comment:
Which one? Http-client? 
I want to get rid of it and use joyful Retrofit everywhere. Right now, Lots of API calls depends on `google-http-client-gson` unfortunately.